### ISSUE TITLE
Install StereoTool seems to be impossible

### ISSUE THREAD

**Issue by SanderCN** [AzuraCast/AzuraCast#6620]
### Installation Method

Docker Installation

### AzuraCast Release Channel

Rolling Release Channel

### Current AzuraCast Version

Rolling Release #5f9f2f7 (2023-09-15 3:20)

### What happened?

My old AzuraCast installation was broken and so I wanted to reinstall it to bring myself to the latest version in a fresh container.
This all worked fine except for Stereo Tool which is a dealbreaker for me. It doesn't matter if I install it in a new VM or the same old one it just doesn't upload.
I always get this error: Uploaded file could not be moved to /var/azuracast/servers/stereo_tool.
It doesn't matter if I upload the command line version, the full generic plugin zip or the DLL inside of it. It just doesn't upload.

### Relevant log output

```shell
azuracast          | [2023-09-16T09:53:13.879792+00:00] AzuraCast.ERROR: Uploaded file could not be moved to /var/azuracast/servers/stereo_tool {"file":"/var/azuracast/www/src/Service/Flow/UploadedFile.php","line":131,"code":0} []
```

**Comment by SanderCN** [comment]
Oh yeah good to also note, i've tested both the latest Stable release and Rolling Release

**Comment by CyberMixRo** [comment]
Any solution on this issue?

**Comment by CyberMixRo** [comment]
Any idea about this issue? Some say they used, but anything ive try have no result, can not be uploaded

**Comment by PeWe79** [comment]
I think this problem is Stereo Tool latest version is v10.10, but on AzuraCast detail vesion suport is v10.00 (StereoTool/PostAction).

**Comment by BusterNeece** [comment]
After investigating further, we've found and fixed the problem in the latest Rolling Release version of AzuraCast that's deploying right now (and should be available for download in about 10-15 minutes after this post).

Basically, when Thimeo released version 10.10 of their Stereo Tool plugin, they packaged it in a completely different format than the original 10.00 version (which was the first version they released in this plugin format). Our code had to be manually updated to support this new version's format, which it now is.

Our code changes have hopefully made the uploader a little more robust to possible future changes or new version releases. Provided their formatting stays largely the same as it is now (just with new versions), we'll be able to support those without code changes being required.

**Comment by RobertC85** [comment]
> After investigating further, we've found and fixed the problem in the latest Rolling Release version of AzuraCast that's deploying right now (and should be available for download in about 10-15 minutes after this post).
> 
> Basically, when Thimeo released version 10.10 of their Stereo Tool plugin, they packaged it in a completely different format than the original 10.00 version (which was the first version they released in this plugin format). Our code had to be manually updated to support this new version's format, which it now is.
> 
> Our code changes have hopefully made the uploader a little more robust to possible future changes or new version releases. Provided their formatting stays largely the same as it is now (just with new versions), we'll be able to support those without code changes being required.

do you have an ETA for that being in the stable release?
